addons/parley/graph_operations/delete_operation.gd

```diff
@@ -0,0 +1,31 @@
+# DeleteShortcut.gd
+extends GraphOperation
+class_name DeleteOperation
```

Comment:
For all of these class_names, could we prefix with Parley to avoid clashes with other plugins? E.g. `ParleyDeleteGraphOperation`